app/controllers/report_controller/saved_reports.rb

```diff
@@ -142,7 +142,7 @@ def get_all_saved_reports
       @settings.store_path(:perpage, :list, @items_per_page) # Set the per page setting for this gtl type
     end
 
-    @sortcol = session["#{x_active_tree}_sortcol".to_sym].nil? ? 0 : session["#{x_active_tree}_sortcol".to_sym].to_i
+    @sortcol = session["#{x_active_tree}_sortcol".to_sym].nil? ? ReportController::DEFAULT_SORT_COL : session["#{x_active_tree}_sortcol".to_sym].to_i
```

Comment:
```suggestion
    @sortcol = session["#{x_active_tree}_sortcol".to_sym].nil? ? DEFAULT_SORT_COL : session["#{x_active_tree}_sortcol".to_sym].to_i
```

Since we're within the same controller, you should need to fully-qualify here